docs/pages.yml

```diff
@@ -10,15 +10,19 @@
     - HPC Cluster: 'content/getting_started/cluster.md'
     - Virtual Machine: 'content/getting_started/virtual_machine.md'
     - Manual Installation: 'content/getting_started/manual_installation.md'
-- Documentation:
-    - Create an App: 'content/getting_started/create_an_app.md'
-    - Framework:
-        - Systems: !include content/framework/systems/pages.moose.yml
-    - Modules:
-        - Phase Field:
-            - Systems: !include content/modules/phase_field/systems/pages.moose.yml
-        - Contact:
-            - Systems: !include content/modules/contact/systems/pages.moose.yml
-    - Documenting Applications:
-        - Creating Documentation: 'content/doc_generation/Overview.md'
-        - Moose Flavored Markdown: 'content/doc_generation/MooseFlavoredMarkdown.md'
+- Application Development:
+    - Create an Application: 'content/application_development/create_an_app.md'
+- Framework:
+    - Systems: !include content/framework/systems/pages.moose.yml
+- Modules:
+    - Phase Field:
+        - Systems: !include content/modules/phase_field/systems/pages.moose.yml
+    - Contact:
+        - Systems: !include content/modules/contact/systems/pages.moose.yml
+- Utilites:
+    - Documentation:
+        - Creation: 'content/utilities/documentation/create.md'
+        - Moose Flavored Markdown: 'content/utilities/documentation/moose_flavored_markdown.md'
+- Help:
+    - Frequently Asked Questions: 'content/help/faq.md'
+    - Contact Us: 'content/help/contact_us.md'
```

Comment:
When serving with 'themes', we start to collide with our Nav bar:

<img width="1304" alt="screen shot 2016-08-26 at 10 54 23 am" src="https://cloud.githubusercontent.com/assets/6414286/18013503/8a327f36-6b7b-11e6-9589-fe73ba1272d0.png">


Looks like we are going to need to trim back a few things.